kratos/geometries/line_2d_2.h

```diff
@@ -897,7 +897,8 @@ class Line2D2 : public Geometry<TPointType>
         // Project point
         const double Tolerance = 1e-14; // Tolerance
 
-        const double Length = this->Length();
```

Comment:
@loumalouomega to avoid the virtual function call you should only remove the `this->` from the function call and duplication is not needed!